### DISCUSSION TITLE
Generate a three-level mosaic

### DISCUSSION THREAD

**Discussion by Mikewangkang** [neperfepx/neper#741] [Q&A] [answered]
![1693397910887](https://github.com/neperfepx/neper/assets/139696590/8bb2f9c7-e7e0-4558-93cd-abd525e0dec8)
![1693398521176](https://github.com/neperfepx/neper/assets/139696590/18341c75-8089-43e7-8290-f311ae491b54)
![1693398521172](https://github.com/neperfepx/neper/assets/139696590/92a4dde7-3c70-4a39-ac1d-d9c5c96cfdb8)
Hello everyone
I want to use neper to generate the effect shown in the figure, specifying some grains for three-layer mosaic, the second layer of mosaic divides the grain into three clusters, and the third layer of mosaic divides each cluster into slats, which can be changed The slat width, and the slat orientation, I would appreciate it if you could help me
best wishes
wang kang

**Comment by rquey** [comment, marked as answer]
Here is an example that should get you on track:
```
neper -T -n 4::3::from_morpho -morpho "voronoi::voronoi::lamellar(w=0.1)" -o test
neper -V test.tess -datacellcol "scaleid(1)" -print img
```
![img](https://github.com/neperfepx/neper/assets/2659376/ac761c99-31f5-4ee7-8e36-a2bb66e69f93)
